detection_rules/eswrap.py

```diff
@@ -736,3 +740,103 @@ def delete_ml_job(ctx: click.Context, job_name, job_type, verbose=True):
 
     if verbose:
         click.echo(f'Deleted {job_type} job: {job_name}')
+
+
+@es_experimental.command('create-dnstwist-rule')
+@click.argument('input-file', type=click.Path(exists=True, dir_okay=False), required=True)
```

Comment:
can you drop a file or dump of an exampled dnstwist file here @ajosh0504 @threat-punter

Reply:
Ugh GitHub won't let me upload a json here but here's a dump of an example json file:
```
[
    {
        "dns-a": [
            "34.107.161.234"
        ],
        "dns-aaaa": [
            "2600:1901:0:1f6d::"
        ],
        "dns-mx": [
            "alt1.aspmx.l.google.com",
            "alt2.aspmx.l.google.com",
            "alt3.aspmx.l.google.com",
            "alt4.aspmx.l.google.com",
            "aspmx.l.google.com"
        ],
        "dns-ns": [
            "ns-1168.awsdns-18.org",
            "ns-1737.awsdns-25.co.uk",
            "ns-339.awsdns-42.com",
            "ns-785.awsdns-34.net"
        ],
        "domain-name": "elastic.co",
        "fuzzer": "original*"
    },
    {
        "dns-a": [
            "184.168.221.30"
        ],
        "dns-mx": [
            "alt1.aspmx.l.google.com",
            "aspmx.l.google.com"
        ],
        "dns-ns": [
            "pdns1.cscdns.net",
            "pdns2.cscdns.net"
        ],
        "domain-name": "elastica.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticb.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticc.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticd.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elastice.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticf.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticg.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elastich.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elastici.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticj.co",
        "fuzzer": "addition"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "192.64.119.33"
        ],
        "dns-mx": [
            "eforward1.registrar-servers.com",
            "eforward2.registrar-servers.com",
            "eforward3.registrar-servers.com",
            "eforward4.registrar-servers.com",
            "eforward5.registrar-servers.com"
        ],
        "dns-ns": [
            "dns1.registrar-servers.com",
            "dns2.registrar-servers.com"
        ],
        "domain-name": "elastick.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticl.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticm.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticn.co",
        "fuzzer": "addition"
    },
    {
        "dns-a": [
            "217.160.0.22"
        ],
        "dns-aaaa": [
            "2001:8d8:100f:f000::2b9"
        ],
        "dns-mx": [
            "mx00.1and1.it",
            "mx01.1and1.it"
        ],
        "dns-ns": [
            "ns1021.ui-dns.biz",
            "ns1021.ui-dns.com",
            "ns1021.ui-dns.de",
            "ns1021.ui-dns.org"
        ],
        "domain-name": "elastico.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticp.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticq.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticr.co",
        "fuzzer": "addition"
    },
    {
        "banner-http": "Apache/2.4.25 (Debian)",
        "dns-a": [
            "103.224.212.220"
        ],
        "dns-mx": [
            "park-mx.above.com"
        ],
        "dns-ns": [
            "ns1.above.com",
            "ns2.above.com"
        ],
        "domain-name": "elastics.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elastict.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticu.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticv.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticw.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticx.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "elasticy.co",
        "fuzzer": "addition"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "107.22.166.14",
            "52.72.59.17",
            "54.205.13.34",
            "54.89.39.4"
        ],
        "domain-name": "elasticz.co",
        "fuzzer": "addition"
    },
    {
        "domain-name": "dlastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "glastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "alastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "mlastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "ulastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "emastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "enastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "ehastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "edastic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elcstic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "104.243.45.179"
        ],
        "dns-ns": [
            "ns1.chookdns.com",
            "ns2.chookdns.com"
        ],
        "domain-name": "elestic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elistic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elqstic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elartic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elaqtic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elawtic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "185.107.56.209"
        ],
        "dns-ns": [
            "ns1.hostresolver.com",
            "ns2.hostresolver.com"
        ],
        "domain-name": "elactic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "ela3tic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elasuic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elasvic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elaspic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elasdic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elas4ic.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elasthc.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastkc.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastmc.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastac.co",
        "fuzzer": "bitsquatting"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "192.64.119.103"
        ],
        "dns-mx": [
            "eforward1.registrar-servers.com",
            "eforward2.registrar-servers.com",
            "eforward3.registrar-servers.com",
            "eforward4.registrar-servers.com",
            "eforward5.registrar-servers.com"
        ],
        "dns-ns": [
            "dns1.registrar-servers.com",
            "dns2.registrar-servers.com"
        ],
        "domain-name": "elastyc.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastib.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastia.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastig.co",
        "fuzzer": "bitsquatting"
    },
    {
        "banner-http": "Apache",
        "dns-a": [
            "169.59.0.130"
        ],
        "dns-mx": [
            "alt1.aspmx.l.google.com",
            "alt2.aspmx.l.google.com",
            "aspmx.l.google.com",
            "aspmx2.googlemail.com",
            "aspmx3.googlemail.com"
        ],
        "dns-ns": [
            "ns1dhl.name.com",
            "ns2jqz.name.com",
            "ns3jnr.name.com",
            "ns4htz.name.com"
        ],
        "domain-name": "elastik.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "elastis.co",
        "fuzzer": "bitsquatting"
    },
    {
        "domain-name": "xn--lastc-u0a65u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa6z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra6a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-i0a99s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-r4a32d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra16f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa92l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa22w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra6e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta92k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-kxa62r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mq1byc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa52b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb43f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa98n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-01c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a7843c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra28k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-lya4114c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2rab.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa65v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb2923c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb74j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-vcb61q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa42a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-h9b8781c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa1183c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-lwb47j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra55r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa0793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9hc6v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-h9b3252c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a6143c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa58x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-7cb12o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-fjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-9hc6w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa86k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-msa7v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tcc6t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-fl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nra3y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra95s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-wdb9253c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a60c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb19p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-ps1bx1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb84p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa03s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa7g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a0033c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya5w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-yeb87m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-hoc6n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra3393c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a5x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-hza83f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-5za69j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a09a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa01w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra17f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-os1bww.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa43t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-mra34y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-m6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a2t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yeb49q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa5893c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa2f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-3wa91w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra86s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-wsa19x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-1jc6141c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-je1br7c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa6x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-m6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra44n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-je1b06c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-zwb59h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-hoc1221c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra52g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-tcc98b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-3ra8x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa82t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa96a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-36b7q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra08f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra6454c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-m51bue.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra15s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-wdb12n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-3sa0r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa8074c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra8104c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za7863c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra3y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-mqb7v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra9c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-vua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-t9b6821c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-kxa22b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-7nck.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa06g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa23i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa08b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra5304c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra78b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-p9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-p3a57o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-8xa89q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-gsa4w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa6o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa8423c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa81g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa6043c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa1893c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa9y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza14k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-xdb9y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-fjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a67a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa45z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a19a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-7nc5481c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza21v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa70a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-fl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta01f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa76y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a93h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza50v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb0813c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-zwb7132c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "clastie.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-ucc0442c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-xsa5u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa5054c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-u9b2252c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-tza80v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya1144c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra1204c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yeb95j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa92q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa27d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa0873c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-06bd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-0k8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa71a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra51y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mqb7752c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa5283c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa5993c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra24n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra31y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra6e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-e81bd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya19o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra09a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa9x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa35y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa11a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--castic-3db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-k7a11q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa74z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-3db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a8853c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa16f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb3x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb73k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yq1byc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-vcb1753c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa3f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-g9b7042c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa2004c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa9x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra78t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa2y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra43c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa78d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--east1c-i2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa6a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zwb8832c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-vcb53o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hray.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa8k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa82n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-o1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-3qa58d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa68i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a36u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa0i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "elastlc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa5w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra4693c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta35f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra10l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-hsc3411c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra22c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra4b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--east1c-3db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-wdb5963c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa3b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a53m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta8783c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-tza92a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-2sa50d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-je1bt4b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb94e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a80p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a4q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-j11bwg.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-01c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-okb54k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta9y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-zsad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-wdb39n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-fjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa6z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra65f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa03q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa0704c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa4893c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra48s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa7z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb8673c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra55c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a5153c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-hoc5691c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a02m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa2154c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-13a55r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa80w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-i6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a34w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa18x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a89n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-4ncd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa4573c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa43s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "elast1e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa0434c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa51u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-k11bsl.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-mqb29f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa5d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-swa5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-0va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-1jc7n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra32f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza45o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa28n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb02f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-wdb9852c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa82d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra210a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa8j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa79a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-q81bd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za8l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-i0a5304c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa76f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mqb3652c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-lya08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb47c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastie-3db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra8664c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra86f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa5963c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa7t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-ds1bo2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa6m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa96c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a55t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwb34b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa4b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-fta07x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta27m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-mqb0103c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-ucc87b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb50j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-7cb20q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-36b10c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-je1bxv.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa06q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza4463c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-yeb13p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-9ra61y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya6g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa6504c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa7p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-4cc8961c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa5d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-lwb93c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa08t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra60g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta5x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa1183c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-xq1bts.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa16k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2raw.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb0553c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra84a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-9qa87d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nra5v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-5za49u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-1nc3521c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa81b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-3sa8t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-okb62h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa57f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta5m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za77c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a91b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-7va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-q4a55q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra6q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra31c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a28r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nra85i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-14a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-u9b8052c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-1jc3741c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta71a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-gsa52i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra04f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-t9b6f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a60a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hra6x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-os1brp.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9db69k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastlc-i2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra88r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa4634c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa97i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa22l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa2e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-1nc4231c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya0b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-vcb35l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-okb32i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa9344c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "eiastlc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-rwa33w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra2d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa02t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a57r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra81n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya20q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra9404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-mwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra14t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza6434c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-t9b2932c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "clastlc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za0q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta9493c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra84t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4wa5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a4404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa8c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tcc0132c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa25s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa61q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa48v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra84r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tza01k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-7va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa06i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-t9b8g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-wxa98k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za62o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-mqb47i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa71g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "clastic.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-t9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa0744c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa5963c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa65d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-5za31t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa6c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa7464c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa29k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "elast1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra92c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra2004c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa69n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yeb05n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "e1astie.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-8xa29t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa6g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa46x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-4cc66b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa1393c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-goc9270c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra11g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-3sa6p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-uh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra52n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa3y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa29x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa6544c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-lwb0932c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra27a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra57n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdbo.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa35g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-i0a69e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdb6613c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-8xa57k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a26a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa7g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa65r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb06c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-lwb4042c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra8793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-n4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-k11b6l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa15l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-g9b33a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a53c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa64q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a9253c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-9qa75y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za6434c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra58s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-m6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa13a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa11n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa0g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra68z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa6m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra08t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-1jc7841c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra4693c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra99x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa75f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-m8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-lya48q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa16q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa40a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa2954c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-t9b6222c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra3664c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb4e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb68d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a87b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb33j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa58c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za3763c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra6104c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za31k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa0p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-hza7704c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-hza74q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra7493c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra84c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra51l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa4i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa84c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa29x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-q4a01i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa83a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a02a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa90t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tza33q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-k7a49g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa44d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa9e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta30a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-9db78j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4wa43g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-p3a79q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9hc9551c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a52h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-fta19c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-5za9404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb85k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-g9b3i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-kwb95f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa84t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-hoc1591c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta1944c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-mqb0702c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-2rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa4293c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-2sa48x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-xsa3q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra23g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa9954c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-8xa8214c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa0z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa05k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra6a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa4i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa0873c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-fl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-rwa45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra93y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-hza22v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-msa3u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra45f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-36b28d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa0i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa8d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa03d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a7u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa63x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-1jc2041c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza26c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-1nc6470c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra9004c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-okb00c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-n4ad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-q4a9204c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2rai.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra60c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb77e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-yq1bz4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta71v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a1053c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-t9b3971c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra77s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa07r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa13v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-hoc2921c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb50q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa85d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-gvc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa9983c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya81i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta5833c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a9253c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-0k8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa4b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb6353c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa11b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa91s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-5za01f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-zeb6u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-7va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-p3a96r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a87n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya99c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa0744c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-q4a9i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-gsa0v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-36b7931c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-36b1691c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa1664c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-4cc9671c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa12n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza1073c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-cs1bnx.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-lwb9132c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-1nc9321c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-36b7332c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta28e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa14r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdb2513c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tcc4021c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya2573c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-mq1bq5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-4cc4142c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa28e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za29b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa60l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa07v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a46n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa43t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra910a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastie-i2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-13a16o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa8813c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-13a83i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb2873c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa79a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-t9b09a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa30g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra5304c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb02p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-5za51a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra80y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa51w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa3604c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra3f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra40g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-msa5r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa8c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-q4a72r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra66s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa61d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra1074c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "eiast1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya87k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-0jc0190c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa10e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-u9b35c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a94u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra47r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-13a37o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa26a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-9ra1x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-1ncr.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa96a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-7nc3031c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra8204c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya24d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa73s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa2404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa0o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nra9w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa7193c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-0k8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-13a7504c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-yq1bq5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-xqa29d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra75r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-yncd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-ucc17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-w8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a84a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a4k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a94c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa3z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-zwb41g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa54s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra98b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-y3ad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-7cb72l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wcb13a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra7a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-fl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza6o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa30f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa66t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra0443c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-zwb32c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa51e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra29s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za2063c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya17c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya69q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa14c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za48u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-u0a28e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra79a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa64i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa4q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-w1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-u3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya2f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa0464c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra24w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta52f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb5283c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yeb6843c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-bjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-0va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta3693c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-0va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra99f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra5643c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa36v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb89p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hra0z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa4924c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra07s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza56p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-kxa30l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4wa3d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-9db1803c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-okb1982c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mqb81m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsc2761c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza43c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb65p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-13a1983c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a1733c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9hc1291c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa01r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-gsa90y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa05p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra4964c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra7764c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a15n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-tza42f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwbq.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-cs1b5p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-13a63t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a4753c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya94v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-t9b81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a4034c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-2ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-p3a78o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-mqb55k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-9ra73d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-kwb7281c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa56z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-goc5381c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-m8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-3wa04b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-mra46d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-7db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa58c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-bra69f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-wdb67e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa2e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-t9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a5463c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-zwb85j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a31v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-h9b57c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-01c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "c1astic.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa5x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa5854c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-0jc5p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb31p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a92w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa57q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa2023c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za22p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-gsa2s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza0363c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-p3a1n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta5234c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-4cc93d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "elastie.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza0363c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra23y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-hsc2081c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-3sa4s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta98v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb3583c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-0va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa98e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza4734c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra09v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-u0a66u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a42l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msal.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza6o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-mwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-36b9r.co",
        "fuzzer": "homoglyph"
    },
    {
        "banner-http": "openresty/1.13.6.1",
        "dns-a": [
            "173.255.194.134",
            "198.58.118.167",
            "45.33.18.44",
            "45.33.2.79",
            "45.33.20.235",
            "45.33.23.183",
            "45.33.30.197",
            "45.56.79.23",
            "45.79.19.196",
            "72.14.178.174",
            "72.14.185.43",
            "96.126.123.244"
        ],
        "dns-mx": [
            "mail.mailerhost.net"
        ],
        "dns-ns": [
            "ns1.mytrafficmanagement.com",
            "ns2.mytrafficmanagement.com"
        ],
        "domain-name": "e1astic.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9db0863c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa9404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa6g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zwb4732c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa99x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa24d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza50d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa9743c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra60y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb0d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-wxa78v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8cb71a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-i0a50q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa37d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-3ra93i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a76f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra72c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-9ra3u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa3273c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-kwb21i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a3453c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-3ra6t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-wxa80b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za60c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa52w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza4r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasic-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra26c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa3943c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-gta2o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa6843c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-wxa60u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--castic-i2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tccm.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza89u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza02c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa5174c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-ucc7671c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-wva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza2n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a2743c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zeb26i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa6093c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-9db4153c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4war.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa49t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9db60n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-8ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-swa84g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa3544c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa72c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra8074c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra7c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta86e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasti-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9hc8122c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a36c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa51q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa7793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa6434c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa5893c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa80u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-lq1bp0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-gra05y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra38a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-zwb3032c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a11h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa68m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za35h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb4913c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya3s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza72k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a87b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa2l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-m6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa620a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa40n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa77v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb4913c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gsa82w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-u0a58s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9hc5451c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra36f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-3ra32y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa4c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-lya87t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza0634c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra82f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra9d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-1nc2581c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-zwb23d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa01a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-nva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-i0a08u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa4564c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-okb89m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa040a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra1754c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa5w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-mq1b45a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra67a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra35s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa2983c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa2274c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-9ra9s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa30w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa0473c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-vcb36n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra93y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa5244c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa8c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-u0a1204c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa69e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-0k8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza6163c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-lwb45c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-vcb7373c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za43c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa12f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa90s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "elastle.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa9073c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa77i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa59r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa13f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra99s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-81a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa10q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza58w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za6163c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a84a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa1v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa6673c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-i2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-3qa46y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra5993c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa23g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb0183c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "ciastic.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-h2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-8xa37v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-9db87n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa12t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa93f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a4133c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa2y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-j7a8082c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa6123c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-4cc67b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-okb7v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa03d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a02c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-3sa99h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb88m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra3b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-swa3d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra4964c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-u0a7y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa5244c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa51c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra43a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra14a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiatic-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-mwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa70f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya0824c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-hocd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa30a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-13a1592c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa11w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa9993c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa09a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a36a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-j7a4193c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-okb3132c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb09i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa46p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa4j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa94f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa330a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "e1ast1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-23a15d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-4cc4861c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-swa19i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa83a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-i8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-lya95v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa2f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-gta6p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra8543c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-7va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-7nc7131c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a8l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta1w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra02c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-msa9s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb03g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra8r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-p3a1704c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-ds1b22a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-q4a80t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-8k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-ctad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-ucc1871c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a6o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-mqb20i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yq1b7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra55a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-5za75w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra8734c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa39m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb48p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-83a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a15e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra03a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-kxa10w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa0i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza02c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa2573c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9hc4741c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa7a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-7nc7o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa31d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-hsc8871c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra0834c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya4c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a90g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-4cc1312c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-9db22r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-7cb40f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-ucc6961c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-okb7782c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-kxa02u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb36p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a84a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa11b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a50b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7ad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "eiastie.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-gta4s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa9274c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-okb18m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra4654c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa50p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-llb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a9563c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa8034c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra72n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-5ib.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-hoc5321c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-g9b1031c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-xsa60i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza7863c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa83r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza5173c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-xsa7r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra27v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa72s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra49b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa42f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-hza34a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-iwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra44a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a4443c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9raz.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-je1bzy.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra1r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a29b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra7p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa9n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra85a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwb10i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za77u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa64g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a21o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya8373c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa42a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza8s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa4323c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a39n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa10w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a8134c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa52a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra5493c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9hc5e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastl-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya7t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa1863c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-tsad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra0864c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa24f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsc6650c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa1793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa0193c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-ucc0171c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9hc9u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza19c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa2743c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9hc92b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a17u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra97z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-m8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-m8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa8j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra11a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-kwb3392c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tza72t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra53w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya1863c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa39i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-vcb1362c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-kxa6514c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-mwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a43a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra0c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-3ra4w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-m51b8e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-1nc4p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra0g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-j7a04p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra56f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra1793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-swa7e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa4973c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-fl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa67a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-sqa63l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa6504c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdb23c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-36b5i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa50t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-h9b86c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-h9b7352c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a9563c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-4cc5z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa55c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-hza05p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra0s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-0va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-kwb94d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hra2w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa8c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-p3a5093c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra73f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-beb4x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa34k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa6273c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-q4a54o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-h9b4681c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-2ra52y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a4k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa52g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-t9b9771c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za73p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-wdb47p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-zwb02d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-7nc2321c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-hoc6031c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa5d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asic-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-hza42k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta33z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a2063c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra86n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasic-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3wa0h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra98f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-gsa27z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-xoc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra4293c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "eiastic.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa28f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-wxa21r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-23a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-fjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a46b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-2ra88z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-k7a7473c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa9334c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-36b3052c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa1074c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra61a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza63p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "clast1c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-45b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa89s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-swa9b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-wdb99k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a97n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra7a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa72t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa35y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-rqa87y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb6973c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-7nc6421c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yeb33e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa06y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-zwb83c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-2sa9933c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-i0a10a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-p3a05t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra26a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-u0a49o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-xta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a32p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-i0a8853c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-ps1bo2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mq1bpd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-wxa2414c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra5b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4wa73l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-beb14p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza1073c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-yeb6452c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-90a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa98f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa43r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-gta0r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa66b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-9db48k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa14c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-gsc6051c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-9ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta1l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra44w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-xqa17y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa26a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-xq1byz.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clstic-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa37b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yqa31t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a2n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra4564c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-4cc8242c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-lya16k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yeb3503c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsas.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-i0a3763c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-u0a19p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-1jc2312c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a8543c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa8754c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-gta58h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb7683c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-fta31s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra8393c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-5za91q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza36c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa6673c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-yeb65k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya78p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa06y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a2334c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-l17b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-t9b43e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa40w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa41f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-36b4091c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwb3603c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra74f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra27i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-3va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-yqa9b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa38a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa02t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatic-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza76h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a58j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-okb53i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra18r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-36b54f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa6g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-j7a03n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-je1b6u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-13a38q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wsa3193c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-lmc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa1144c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra5764c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-g9b50b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-wk8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza84c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-8xa17c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-o3a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdb61k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa01g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-mwa.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa81w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-9uc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa9344c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastlc-3db.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-vcb14l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-gva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-7cb94n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa38c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra7864c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-9ra7v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-2sa5m.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb8f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-msa00f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra10c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra7z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-q4a33o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra96a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya8d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa33p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-7nc9761c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatlc-kz7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-zwb0181c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsaz.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa42r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasie-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta2983c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra80g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa67c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-mqb6713c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza17w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-xdb6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa64y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-9db7603c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-tza3604c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-1nc6861c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-p3a8543c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a90p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za2n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-9db95p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-21a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra28f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-fjc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-lwb3332c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa40a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-hza14t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-hra7593c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-5za2334c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-q4a3292c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-m3ad.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-8cb44k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-yeb9303c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra41f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-4cc95b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra79v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nrar.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-1jc75a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-jta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa2l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-4ta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-gsa78b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-hsc9211c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-x0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-h9bo.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa55a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstlc-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msa80c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-1jc6412c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-yeb2563c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-k7a5833c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza0634c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-swa4844c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-uva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-wcb3583c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-h4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-8xa65d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa09a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4wa9b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-zwb24f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-msa81i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-hza0q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-bra93g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-13a99u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-m8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-msa32d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-k7a71n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-wwbd.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra8a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1stic-cxc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-cua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra53f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza19u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-94a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb1183c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastlc-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-msa58f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-wxa91p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-36b0981c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra52y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clatic-kjb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa43c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa31d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-t9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-okb0282c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-lq1b7s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-2ra64d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa40b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-rqa99d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-8rb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mrat.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a99j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa12w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--last1c-ova.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa48d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-wdb11l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-2ra05t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-u9b06c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra9w.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4qa60t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9hc4022c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-u0a86j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb6033c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-mra4b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-5y7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-8cb9383c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa8793c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-9ra23i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hra56i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra64y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiasti-g1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa47f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra48k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa3f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9ra57k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya24v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-vcb81f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa5174c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya22x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-7va.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa2993c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya75c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-lya7934c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa67a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-zwb6192c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-0jc6102c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa2234c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa0g.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-5za94c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatie-5lc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra08b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-k7a0p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2ra26t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-4qa6e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa46f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-fta70t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rqa38a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra0593c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-7cb7162c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-bra5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eatic-g9b68c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3qa10u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa02l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-sqa1i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elat1c-56c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-13a7l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eiastc-t9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-nra7z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-36b0h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra23y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-q3a56d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-mqb08f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-gra17d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-nqb44k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-7nc9370c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-wsa64f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-bra02c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra2d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-gsa03d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4qa19s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-q4a19a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-7cb3273c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-oh8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elas1c-skb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a63n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-1ua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-msa20y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-sqa57f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast1-n2a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-q4a3683c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-bl8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-4wa7973c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-8xa1v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-9db4752c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-36b5791c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a8134c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9ra1734c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-mqb38e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-01c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-u0a0j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-p3a33b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra5c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-1jc4o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-okb9143c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-hra9d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-nra79f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-71c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaic-wcb8023c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-8xa2f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-nra16n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-fta89r.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-k7a80c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa83v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-je1b8x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra12f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--estic-hoc8661c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra45f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-ucc6242c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwb49h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa8893c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9hc8841c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-gpc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-p3a25i.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya4713c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-nra500a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-swa59x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra80l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-tza04o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-fta59s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-wxa07d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasti-g5a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-1nc0131c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-okb14h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-hra40a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra53n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-zwb0572c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-i0a07u.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstie-xc8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-01c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa3o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1atic-dkb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-wsa21d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-ucc15d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-lwb9403c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-p3a5692c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastic-31c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-lya60t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-2sa21s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastie-2k7b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-3ra4q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clasic-6yb.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-gra0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-3ra0v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaste-t9a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-13a59f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--1astic-9oc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa20s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa01a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-9db16e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstic-j0a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elastc-m2c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easic-t9b17c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eistic-qta.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra30c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1asti-u1a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-m6b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-msas.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-okb71l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza77c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elaslc-607b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-q4a0343c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-tza15p.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa72d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-okb3523c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-2ra13c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-9qa84a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa61t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-i0a28j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-xqa40b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa14d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-kxa19v.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-msa7534c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-2ra4004c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-sqa0z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "e1astlc.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-9ra60y.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-hza18h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa99c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-4cc5571c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-lya7663c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elatc-xdb29e.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--e1astc-0k8b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsic-hra2404c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-u0a12c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-2rap.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-okb6082c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lstic-rwa4534c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-wxa8d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-36b92b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-okb16o.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-gsa6t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-9ra44s.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastic-hva.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-wsa2n.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elst1c-jua.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-yeb86k.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elstc-yqa4h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-wsa6z.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--astic-3ra44d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-g9b1h.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--easti-8xa49a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elast-xsa1t.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasti-kxa4j.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-tza60c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-7cb7553c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lastc-3ra1664c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-g9b1422c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-3ra87f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-4wa0473c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elsti-4qa2b.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--clastc-08a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-kxa44q.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-3ra1393c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa99c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasi-lya41d.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--iastic-v4a.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eastc-k7a0304c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-wxa05x.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--latic-7cb93l.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elasc-2sa3693c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--eltic-yqa86f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--elati-kxa9073c.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "xn--lasic-gsa29f.co",
        "fuzzer": "homoglyph"
    },
    {
        "domain-name": "e-lastic.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "el-astic.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "ela-stic.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "elas-tic.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "elast-ic.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "elasti-c.co",
        "fuzzer": "hyphenation"
    },
    {
        "domain-name": "elasxtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastgic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastzic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elast6ic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elas5tic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaxstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elzastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastfic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaystic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elpastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elast5ic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastyic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elsastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasrtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "eladstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elawstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastioc.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasti8c.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaestic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasftic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasgtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasti9c.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastkic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elkastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "el2astic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastric.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "ela2stic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "eklastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elast8ic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elas6tic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasetic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasdtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasatic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "ela1stic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elmastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasytic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaqstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastijc.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasztic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "eloastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastikc.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elqastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "eolastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elazstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasqtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elyastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elast9ic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elaswtic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastiuc.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastjic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "emlastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elwastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "el1astic.co",
        "fuzzer": "insertion"
    },
    {
        "banner-http": "openresty",
        "dns-a": [
            "52.58.78.16"
        ],
        "dns-aaaa": [
            "2a05:d014:9da:8c10:306e:3e07:a16f:a552"
        ],
        "dns-ns": [
            "ns1.dan.com",
            "ns2.dan.com"
        ],
        "domain-name": "eplastic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elasstic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastoic.co",
        "fuzzer": "insertion"
    },
    {
        "domain-name": "elastuic.co",
        "fuzzer": "insertion"
    },
    {
        "dns-a": [
            "91.195.240.94"
        ],
        "dns-ns": [
            "ns1cmt.name.com",
            "ns2jrt.name.com",
            "ns3dty.name.com",
            "ns4bht.name.com"
        ],
        "domain-name": "elastc.co",
        "fuzzer": "omission"
    },
    {
        "banner-http": "Apache/2.4.25 (Debian)",
        "dns-a": [
            "103.224.182.242"
        ],
        "dns-mx": [
            "park-mx.above.com"
        ],
        "dns-ns": [
            "ns11.above.com",
            "ns12.above.com"
        ],
        "domain-name": "elasic.co",
        "fuzzer": "omission"
    },
    {
        "banner-http": "openresty",
        "dns-a": [
            "199.59.242.153"
        ],
        "dns-mx": [
            "mx76.m2bp.com",
            "mx76.mb1p.com"
        ],
        "dns-ns": [
            "ns1.bodis.com",
            "ns2.bodis.com"
        ],
        "domain-name": "elstic.co",
        "fuzzer": "omission"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "208.87.129.89"
        ],
        "dns-mx": [
            "mx.yandex.ru"
        ],
        "dns-ns": [
            "deb.ns.cloudflare.com",
            "derek.ns.cloudflare.com"
        ],
        "domain-name": "elasti.co",
        "fuzzer": "omission"
    },
    {
        "banner-http": "openresty",
        "dns-a": [
            "34.102.136.180"
        ],
        "dns-ns": [
            "ns53.domaincontrol.com",
            "ns54.domaincontrol.com"
        ],
        "domain-name": "lastic.co",
        "fuzzer": "omission"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "185.107.56.197"
        ],
        "dns-ns": [
            "ns1.commonmx.com",
            "ns2.commonmx.com"
        ],
        "domain-name": "elatic.co",
        "fuzzer": "omission"
    },
    {
        "domain-name": "eastic.co",
        "fuzzer": "omission"
    },
    {
        "domain-name": "elastiic.co",
        "fuzzer": "repetition"
    },
    {
        "domain-name": "ellastic.co",
        "fuzzer": "repetition"
    },
    {
        "domain-name": "eelastic.co",
        "fuzzer": "repetition"
    },
    {
        "domain-name": "elasttic.co",
        "fuzzer": "repetition"
    },
    {
        "domain-name": "elzstic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaatic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "rlastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elastiv.co",
        "fuzzer": "replacement"
    },
    {
        "banner-http": "nginx",
        "dns-a": [
            "162.222.213.198"
        ],
        "dns-ns": [
            "ns1.hostresolver.com",
            "ns2.hostresolver.com"
        ],
        "domain-name": "elastoc.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elasfic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "epastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "el1stic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "wlastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "zlastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elast8c.co",
        "fuzzer": "replacement"
    },
    {
        "banner-http": "Apache/2.4.25 (Debian)",
        "dns-a": [
            "103.224.182.242"
        ],
        "dns-mx": [
            "park-mx.above.com"
        ],
        "dns-ns": [
            "ns11.above.com",
            "ns12.above.com"
        ],
        "domain-name": "elasric.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elsstic.co",
        "fuzzer": "replacement"
    },
    {
        "dns-a": [
            "173.255.194.134",
            "198.58.118.167",
            "45.33.18.44",
            "45.33.2.79",
            "45.33.20.235",
            "45.33.23.183",
            "45.33.30.197",
            "45.56.79.23",
            "45.79.19.196",
            "72.14.178.174",
            "72.14.185.43",
            "96.126.123.244"
        ],
        "dns-mx": [
            "mail.mailerhost.net"
        ],
        "dns-ns": [
            "ns1.mytrafficmanagement.com",
            "ns2.mytrafficmanagement.com"
        ],
        "domain-name": "3lastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "eoastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaetic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elastif.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaytic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elasgic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elystic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elas5ic.co",
        "fuzzer": "replacement"
    },
    {
        "banner-http": "openresty",
        "dns-a": [
            "52.58.78.16"
        ],
        "dns-aaaa": [
            "2a05:d014:9da:8c10:306e:3e07:a16f:a552"
        ],
        "dns-ns": [
            "ns1.dan.com",
            "ns2.dan.com",
            "verification-c3ccc9c1.ns3.dan.hosting"
        ],
        "domain-name": "elastix.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elastid.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaxtic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elastjc.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaszic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elaztic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elastuc.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "slastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elast9c.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elasyic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "4lastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elwstic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "ekastic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "eladtic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "elas6ic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "el2stic.co",
        "fuzzer": "replacement"
    },
    {
        "domain-name": "e.lastic.co",
        "fuzzer": "subdomain"
    },
    {
        "domain-name": "el.astic.co",
        "fuzzer": "subdomain"
    },
    {
        "domain-name": "ela.stic.co",
        "fuzzer": "subdomain"
    },
    {
        "domain-name": "elas.tic.co",
        "fuzzer": "subdomain"
    },
    {
        "domain-name": "elast.ic.co",
        "fuzzer": "subdomain"
    },
    {
        "domain-name": "leastic.co",
        "fuzzer": "transposition"
    },
    {
        "domain-name": "ealstic.co",
        "fuzzer": "transposition"
    },
    {
        "domain-name": "elsatic.co",
        "fuzzer": "transposition"
    },
    {
        "domain-name": "elatsic.co",
        "fuzzer": "transposition"
    },
    {
        "dns-ns": [
            "ns1.hover.com",
            "ns2.hover.com"
        ],
        "domain-name": "elasitc.co",
        "fuzzer": "transposition"
    },
    {
        "domain-name": "elastci.co",
        "fuzzer": "transposition"
    },
    {
        "banner-http": "openresty/1.13.6.1",
        "dns-a": [
            "173.255.194.134",
            "198.58.118.167",
            "45.33.18.44",
            "45.33.2.79",
            "45.33.20.235",
            "45.33.23.183",
            "45.33.30.197",
            "45.56.79.23",
            "45.79.19.196",
            "72.14.178.174",
            "72.14.185.43",
            "96.126.123.244"
        ],
        "dns-mx": [
            "mail.mailerhost.net"
        ],
        "dns-ns": [
            "ns1.mytrafficmanagement.com",
            "ns2.mytrafficmanagement.com"
        ],
        "domain-name": "elastec.co",
        "fuzzer": "vowel-swap"
    },
    {
        "domain-name": "elostic.co",
        "fuzzer": "vowel-swap"
    },
    {
        "domain-name": "olastic.co",
        "fuzzer": "vowel-swap"
    },
    {
        "domain-name": "elustic.co",
        "fuzzer": "vowel-swap"
    },
    {
        "domain-name": "ilastic.co",
        "fuzzer": "vowel-swap"
    },
    {
        "domain-name": "elasticco.co",
        "fuzzer": "various"
    },
    {
        "banner-http": "openresty",
        "dns-a": [
            "34.102.136.180"
        ],
        "dns-mx": [
            "mailstore1.secureserver.net",
            "smtp.secureserver.net"
        ],
        "dns-ns": [
            "ns27.domaincontrol.com",
            "ns28.domaincontrol.com"
        ],
        "domain-name": "elastic-co.com",
        "fuzzer": "various"
    }
]
```